packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -688,6 +688,10 @@ class FlutterPlugin implements Plugin<Project> {
         } catch (FileNotFoundException ignored) {
             throw new GradleException("settings.gradle/settings.gradle.kts does not exist: ${settingsGradleFile(project).absolutePath}")
         }
+        // TODO(matanlurey): https://github.com/flutter/flutter/issues/48918.
+        project.logger.quiet("Warning: This project is still reading the deprecated '.flutter-plugins. file.")
+        project.logger.quiet("In an upcoming stable release support for this file will be completely removed and your build will fail.")
+        project.logger.quiet("For an example of updating Android build scripts, see https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply.")
```

Comment:
Is this the right link? I dont see .flutter-plugins anywhere on this page (which I would expect)

Reply:
There is no canonical link for "keep your gradle files up to date", is there?

Reply:
but this isnt "keep your gradle files up to date" i dont think

.flutter-plugins was the way and we changed the way so we should have migration explanation someplace.

Reply:
Ok I can write something.

Reply:
I searched for the documentation and cant find any https://github.com/search?q=org%3Aflutter+.flutter-plugins&type=code

Reply:
https://github.com/flutter/flutter/pull/45379 possibly related pr